apps/ui/src/views/Space/Proposals.vue

```diff
@@ -115,6 +150,71 @@ watchEffect(() => setTitle(`Proposals - ${props.space.name}`));
             }
           ]"
         />
+        <div v-if="space.labels?.length" class="sm:relative">
+          <PickerLabel
+            v-model="labels"
+            :labels="space.labels"
+            :button-props="{
+              class: [
+                'flex items-center gap-2 relative rounded-full leading-[100%] border button h-[42px] top-1 text-skin-link bg-skin-bg',
+                {
+                  'max-w-[230px]': labels.length >= MAX_SHOWN_LABELS_FILTER
+                }
+              ]
+            }"
+            :panel-props="{ class: 'sm:min-w-[290px] sm:ml-0 !mt-3' }"
+          >
+            <template #button>
+              <div
+                class="absolute top-[-10px] bg-skin-bg px-1 left-2.5 text-sm text-skin-text"
+              >
+                Labels
+              </div>
+              <div
+                v-if="labels.length"
+                class="flex mx-2.5 overflow-hidden items-center"
+              >
+                <ul
+                  v-if="labels.length"
+                  class="flex gap-1 mr-4"
+                  :class="{
+                    'mr-[50px]': labels.length >= MAX_SHOWN_LABELS_FILTER
+                  }"
+                >
+                  <li
+                    v-for="id in labels.slice(0, MAX_SHOWN_LABELS_FILTER)"
+                    :key="id"
+                  >
+                    <UiProposalLabel
+                      :label="spaceLabels[id].name"
+                      :color="spaceLabels[id].color"
+                    />
+                  </li>
+                </ul>
+                <div
+                  class="flex gap-1 items-center absolute rounded-r-full right-2 top-[1px] bottom-[1px] bg-gradient-to-l via-skin-bg from-skin-bg"
+                  :class="{ 'pl-6': labels.length >= MAX_SHOWN_LABELS_FILTER }"
+                >
+                  <span
+                    v-if="labels.length >= MAX_SHOWN_LABELS_FILTER"
+                    class="text-skin-link text-sm"
+                    v-text="`(${labels.length})`"
+                  />
+                  <button
+                    v-if="labels.length"
+                    class="text-skin-text rounded-full hover:text-skin-link"
+                    title="Clear all labels filter"
+                    @click.stop="handleClearLabelsFilter"
+                    @keydown.enter.stop="handleClearLabelsFilter"
+                  >
+                    <IH-x-circle size="16" />
+                  </button>
```

Comment:
Not important! Just a small UX thought, when we click on clear button the picker is still open
![Oct-23-2024 08-20-47](https://github.com/user-attachments/assets/dd46bc3a-d9bc-4a87-88b7-01f714f35a46)

User may click it when they don't want any label filters, showing the modal mean they have to click outside one more time to close the picker